settings/src/main/java/org/odk/collect/settings/keys/ProjectKeys.kt

```diff
@@ -67,4 +67,7 @@ object ProjectKeys {
     const val BASEMAP_SOURCE_OSM = "osm"
     const val BASEMAP_SOURCE_USGS = "usgs"
     const val BASEMAP_SOURCE_CARTO = "carto"
+
+    // remembered defaults
+    const val KEY_SAVED_FORM_SORT_ORDER = "instanceUploaderListSortingOrder"
```

Comment:
You should add a new key for `BlankFormListViewModel` too if you decided that it's a better approach than storing a raw string in a view model.